src/features/hotspots/details/HexBadge.tsx

```diff
@@ -28,17 +30,17 @@ const HexBadge = ({ rewardScale }: Props) => {
   return (
     <TouchableOpacityBox
       onPress={handlePress}
-      backgroundColor="purpleMain"
-      borderRadius="s"
+      backgroundColor="grayBox"
+      borderRadius="ms"
       flexDirection="row"
-      justifyContent="space-between"
+      justifyContent="center"
       alignItems="center"
-      paddingStart="s"
-      height="100%"
-      marginLeft="s"
+      width={59}
+      height={30}
+      marginLeft="xs"
     >
-      <Hex color="white" width={14} />
-      <Text color="white" variant="body2Medium" paddingHorizontal="s">
+      <Hex color={greenOnline} width={14} />
```

Comment:
Should we do different colors here depending on the ratio?

Reply:
No, but explorer does https://github.com/helium/explorer/blob/master/components/Hotspots/utils.js#L101

Reply:
Nice, I'll check it out.